packages/mui-material/src/Slider/Slider.d.ts

```diff
@@ -46,8 +46,7 @@ export interface SliderOwnProps {
   /**
    * The components used for each slot inside.
    *
-   * This prop is an alias for the `slots` prop.
-   * It's recommended to use the `slots` prop instead.
+   * @deprecated use the `slots` prop instead. This prop will be removed in v7.
```

Comment:
Let's add the "How to migrate" link to both deprecations, example: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Alert/Alert.d.ts#L61